packages/cli/src/models/initializer/truffle/TruffleProjectInitializer.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'path';
+import { FileSystem as fs } from 'zos-lib';
+import TruffleConfig from './TruffleConfig';
+
+const TruffleProjectInitializer = {
+
+  call(root = process.cwd()): void {
```

Comment:
Pretty sure `root` is a string (or maybe `typescript` has its own `path` type? :stuck_out_tongue:)